onnxruntime/core/providers/cuda/shared_inc/cuda_utils.h

```diff
@@ -33,5 +34,25 @@ std::unique_ptr<IConstantBuffer<T>> CreateConstantOnes();
 template <typename T>
 void Fill(T* output, T value, int64_t count);
```

Comment:
Scott McKay
Name is very generic given its scope.

Sherlock Huang
Moved inside struct TArray